ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -183,15 +206,23 @@ def create_cluster(cluster_name, ip, exclude_ports, base_dir, plugins=None):
         ports = System.get_free_ports(port_range, exclude_ports, 2, client)
         node_name = System.get_my_machine_id(client)
 
+        home_dir = ArakoonInstaller.ARAKOON_HOME_DIR.format(base_dir, cluster_name)
+        log_dir = ArakoonInstaller.ARAKOON_LOG_DIR.format(cluster_name)
+        tlog_dir = ArakoonInstaller.ARAKOON_TLOG_DIR.format(base_dir, cluster_name)
+
+        ArakoonInstaller.archive_existing_arakoon_data(ip, home_dir, base_dir + '/arakoon/', cluster_name)
+        ArakoonInstaller.archive_existing_arakoon_data(ip, log_dir, '/var/log/arakoon/', cluster_name)
```

Comment:
I'd use sth like ```ArakoonInstaller.ARAKOON_LOG_DIR.format('')``` here to allow us to change that path by just updating the global variable